ghga_connector/__main__.py

```diff
@@ -14,3 +14,71 @@
 # limitations under the License.
 
 """Entrypoint of the package"""
+
+
+from os import path
+
+import pycurl
+import typer
+
+app = typer.Typer()
+
+
+@app.command()
+def upload(
+    api_url: str = typer.Option(..., help="Url to the upload contoller"),
+    file_id: str = typer.Option(..., help="The id if the file to upload"),
+    file_path: str = typer.Option(..., help="The path to the file to upload"),
+):
+    """
+    Command to upload a file
+    """
+    if not path.isfile(file_path):
+        typer.echo(f"The file {file_path} does not exist.")
+        raise typer.Abort()
```

Comment:
Could you maybe add some small tests that check that this function fails when passing a non-exisiting file path? You can call the function directly in the test, to need to use the CLI.

Same for the other Abort statements in the upload/download functions.